src/systems/environmental_sensor_system/EnvironmentalSensorSystem.cc

```diff
@@ -98,7 +170,7 @@ class EnvironmentalSensor : public gz::sensors::Sensor
       this->frameId = _sdf.Element()->Get<std::string>("frame_id");
     }
 
-    gzdbg << "Loaded environmental sensor for " << this->fieldName
+    gzdbg << "Loaded environmental sensor for " << this->fieldName[0]
```

Comment:
@arjo129 hmm, the first field may not necessarily be descriptive for vector data.